src/mlpack/bindings/python/print_input_processing.hpp

```diff
@@ -161,12 +174,12 @@ void PrintInputProcessing(
    * # Detect if the parameter was passed; set if so.
    * if param_name is not None:
    *   try:
-   *     MoveFromPtr[Model](CLI.GetParam[Model]('param_name'),
-   *         (<ModelType?> param_name).modelptr)
+   *     SetParamPtr[Model]('param_name', (<ModelType?> param_name).modelptr,
+   *         CLI.HasParam('copy_all_inputs'))
    *   except TypeError as e:
    *     if type(param_name).__name__ == "ModelType":
-   *       MoveFromPtr[Model](CLI.GetParam[Model]('param_name'),
-   *           (<ModelType> param_name).modelptr)
+   *       SetParamPtr[Model]('param_name', (<ModelType> param_name).modelptr,
+   *           CLI.HasParam('copy_all_inputs'))  TODO
```

Comment:
Is this done?

Reply:
Yeah, I wrote that down when I was getting up to get dinner, and when I came back and wrote the code apparently I forgot to remove the TODO... :)